Could not find a publish target with the name: @electron-forge/publisher-github

### Preflight Checklist
<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->

* [x] I have read the [contribution documentation](https://github.com/electron-userland/electron-forge/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Forge Version:**
  * 6.0.0-beta.53
* **Electron Version:**
  * v10.1.1
* **Operating System:**
  * Windows 10 (19042.487)
* **Last Known Working Electron Forge version:**:
  * 6.0.0-beta.52

### Expected Behavior
Command "npm run publish" gets app published on GitHub

### Actual Behavior
Getting error:

√ Making for target: squirrel - On platform: win32 - For arch: x64
× Resolving publish target: @electron-forge/publisher-github

An unhandled error has occurred inside Forge:
Could not find a publish target with the name: @electron-forge/publisher-github. Make sure it's listed in the devDependencies of your package.json
Error: Could not find a publish target with the name: @electron-forge/publisher-github. Make sure it's listed in the devDependencies of your package.json
    at asyncFn (D:\mtgap\node_modules\@electron-forge\core\src\api\publish.ts:170:17)
    at D:\mtgap\node_modules\@electron-forge\async-ora\src\ora-handler.ts:35:5
    at new Promise (<anonymous>)
    at asyncOra (D:\mtgap\node_modules\@electron-forge\async-ora\src\ora-handler.ts:34:10)
    at publish (D:\mtgap\node_modules\@electron-forge\core\src\api\publish.ts:167:13)
    at D:\mtgap\node_modules\@electron-forge\cli\src\electron-forge-publish.ts:36:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mtgaprotracker@2.1.9 publish: `electron-forge publish`
npm ERR! Exit status 1

### To Reproduce
Try to "npm run publish"

### Additional Information
MAKE operation works fine, as well as app launches normally in DEV mode. But publish command fails for some reason. 

package.json looks like this:
"devDependencies": {
    "@electron-forge/cli": "6.0.0-beta.53",
    "@electron-forge/maker-deb": "6.0.0-beta.53",
    "@electron-forge/maker-dmg": "6.0.0-beta.53",
    "@electron-forge/maker-rpm": "6.0.0-beta.53",
    "@electron-forge/maker-squirrel": "6.0.0-beta.53",
    "@electron-forge/maker-zip": "6.0.0-beta.53",
    "@electron-forge/plugin-webpack": "6.0.0-beta.53",
    "@electron-forge/publisher-github": "6.0.0-beta.53", 

and "npm install" worked fine after version increment.

--- a/packages/publisher/github/src/Config.ts
+++ b/packages/publisher/github/src/Config.ts
@@ -1,4 +1,4 @@
-import { OctokitOptions } from './util/github';
+import { OctokitOptions } from '@octokit/core/dist-types/types.d';
 
 export interface GitHubRepository {
   /**

--- a/packages/publisher/github/src/util/github.ts
+++ b/packages/publisher/github/src/util/github.ts
@@ -2,8 +2,6 @@ import { Octokit } from '@octokit/rest';
 import { OctokitOptions } from '@octokit/core/dist-types/types.d';
 import { merge } from 'lodash';
 
-export { OctokitOptions } from '@octokit/core/dist-types/types.d';
-
 export default class GitHub {
   private options: OctokitOptions;
 

